test/testtokenize.cpp

```diff
@@ -8495,6 +8497,24 @@ class TestTokenizer : public TestFixture {
         ASSERT_NO_THROW(tokenizeAndStringify(code));
         ASSERT_EQUALS("void ( * const f ) ( ) ;", tokenizeAndStringify("typedef void (*fp_t)(); fp_t const f;"));
     }
+
+    void simplifyPlatformTypes() {
```

Comment:
Shouldn't these tests go here? https://github.com/danmar/cppcheck/pull/7416

Reply:
They used to be there but I included them here to have more coverage of the various parts of the code which triggered the issue.